### ISSUE TITLE
Python - Parsing data from files (05/18)

### ISSUE THREAD

**Issue by anitabe404** [studydash/cards#46]
## Motivation/Problem Statement 💭❓:
Become a Pythonista!

---

## Today's Goal/Solution 🥅:
Work on Python Data Structures course.

---

## Result 🍓🍌🍉:
I'm almost done with the Week 4 content (I have a quiz to do, but it should be fairly simple).

---

## Observations & Next Steps 🔭👣:
I'm enjoying the course and looking forward to learning more about Python and doing more exercises.
---

## References 🔗:
Link any useful resources you may have found throughout the day.

---

## Extra/Fun (Optional) 🎈🎉🥳:
My mom is in town, so I gotta keep these status updates brief!

**Comment by r002** [comment]
Of course!  Family is the best!  Have fun! 😀

Thanks for taking the time to contribute though!  Even with everything being so busy!  Keeping the streak alive!  There was ~40 minutes left on the clock and both you & Matt were still MIA so I was all 😬😬😬.

Should've never doubted! 😅😌

**Comment by mccurcio** [comment]
Hey you two,
Any suggestions for times for our first group group call? 
Is it possible to set one up this week or should we go next week. 
I am open this week.
Anyone?

**Comment by r002** [comment]
I'd be game for a group call! 😀
This Wednesday or Thursday would work best for me; nearly any time either of those two days.
If not this week, then maybe sometime late next week?
Looking forward to it! 🥳

**Comment by mccurcio** [comment]
Hey Robert,
Thursday sounds good too. Let me ping Anita and see what she says. Or maybe
we will hear from here hear? ;)


Thank you,

Matt Curcio
M: 401-316-5358
W: https://github.com/mccurcio

-----------------------------------------
What you think, you become.
What you imagine, you create.

Gautama Buddha
-----------------------------------------

**Comment by anitabe404** [comment]
I can't meet this week, but you guys are welcome to meet up without me. I can meet next week.

**Comment by mccurcio** [comment]
@Anita Sure, Us 'GUYS' may just meet without you. ;))

@Robert I'm game just ping me...

I have added a link to my second favorite app for on the internet,
[When2Meet](https://www.when2meet.com/?11938272-RgzGU)

Just add your name, (no need for passwords) and block out the best times
you are available.
I was thinking our first meeting could be 15-30 minutes any day except
Saturday (even that is negotiable??) between 10AM & 11PM sound reasonable.

Is it best to shoot for a consistent time maybe starting next every week?
(Which seems easiest overall)

**Comment by r002** [comment]
@mccurcio Cool website!  Thanks for sharing.  I've filled in my available times for this week-- basically, 4-5p ET today or 2-3p tomorrow; lmk which works best for you!  Want to use the same Google Meet url that you shared last time?

@anitabe404 Np!  Sounds like your mom's in town so you're probably busy, busy!  Have fun!  We'll catch you next week! 👍

**Comment by r002** [comment]
@mccurcio Yep, I have your email-- I've sent you an invite for tmr at 2p ET.  We're all set!  See you then!  😀
